internal/controller/comparators/authpolicy_comparator.go

```diff
@@ -27,7 +27,6 @@ func GetAuthPolicyComparator() ResourceComparator {
 		desiredAP := desired.(*kuadrantv1.AuthPolicy)
 
 		return equality.Semantic.DeepDerivative(desiredAP.Spec, currentAP.Spec) &&
-			equality.Semantic.DeepDerivative(desiredAP.GetLabels(), currentAP.GetLabels()) &&
-			equality.Semantic.DeepDerivative(desiredAP.GetAnnotations(), currentAP.GetAnnotations())
+			equality.Semantic.DeepDerivative(desiredAP.GetLabels(), currentAP.GetLabels())
```

Comment:
so, no label customization is possible?
What about labels added by ocp or other operators (if any)?

Reply:
It is possible to add any custom labels by users. It only checks required labels are there or not.